CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,4166 @@
+		name = "Capodistria for aid"
+		
+		YUG = {
+		    any_pop = { limit = { culture = croat OR = { type = soldiers type = officers } } move_pop = 771 #Zagreb }
+		}
+		
+		set_country_flag_flag = ITA_CRO_Koper
+		ai_chance = { factor = 0.45 }
+	}
+
+	option = {
+		name = "No agreement is found"
+		prestige = -25
+		
+		any_pop = { 
+			limit = { culture = croat }
+		    consciousness = 2
+			militancy = 2
+		}
+		any_pop = { 
+			limit = { is_primary_culture = yes }
+		    scaled_consciousness = { factor = 2 ideology = nationalist }
+			scaled_militancy = { factor = 2 ideology = nationalist }
+		}
+		
+		clr_global_flag = ITA_CRO_Ustasha_funding
+		ai_chance = { factor = 0.2 }
+	}
+	
+}
+
+#Italian intervention in the Croatian War of Independence
+country_event = {
+	id = 8020437
+	title = "Italian intervention in the Croatian War of Independence"
+	desc = EVT_8020437_DESC
+	picture = "YUG_CRO_indy_war"
+	
+	fire_only_once = yes
+	
+	trigger = {
+		tag = ITA
+		year = 1989
+		NOT = { has_global_flag = 1992_start_date_flag }
+		
+		government = populist_dictatorship
+		is_vassal = no
+		war = no
+		
+		money = -150000
+		
+		#Croatian War of Independence
+		YUG = { 
+			exists = yes
+			
+			is_vassal = no
+			war_with = CRO
+		}
+		
+		CRO = { 
+			exists = yes
+			
+			is_vassal = no
+			war_with = YUG
+		}
+		
+		has_country_flag = ITA_delle_Chiaie
+		has_global_flag = ITA_CRO_Ustasha_funding
+    	has_global_flag = ITA_Operation_Tora_Tora
+	}
+	
+	mean_time_to_happen = { days = 1 }
+
+	option = {
+    	name = "Support the Croatian Government!"
+		prestige = 25
+		badboy = 2
+		
+		money = -100000
+		
+		#Domestic effect
+		any_pop = { 
+			limit = { is_primary_culture = yes }
+		    scaled_consciousness = { factor = -2 ideology = nationalist }
+			scaled_militancy = { factor = -2 ideology = nationalist }
+		}
+		
+		#Aid to Croatia
+		CRO = {
+			money = 100000
+			small_arms = 1000
+			weaponry = 1000 
+			aeroplanes = 1000
+			ammunition = 1000
+			automobiles = 1000
+			
+			add_country_modifier = { name = post_war_austerity duration = 3650 }
+		}
+		
+		#Gain relations
+		relation = { who = CRO value = 200 }
+		diplomatic_influence = { who = CRO value = 100 }
+		create_alliance = CRO
+		
+		#Lose relations with Yugoslavia
+		random_country = {
+			limit = { tag = YUG is_vassal = no }
+			relation = { who = THIS value = -100 }
+			THIS = { diplomatic_influence = { who = YUG value = -100 } }
+		}
+		
+		#Lose relations with Slovenia
+		random_country = {
+			limit = { tag = SLO is_vassal = no }
+			relation = { who = THIS value = -50 }
+			THIS = { diplomatic_influence = { who = SLO value = -50 } }
+		}
+		
+		ai_chance = { factor = 0.95 }
+	}
+	
+	option = {
+    	name = "There is no need to interfere"
+		relation = { who = CRO value = -50 }
+		diplomatic_influence = { who = CRO value = -50 }
+		ai_chance = { factor = 0.05 }
+	}
+
+}
+
+#Muggia agreement - 1
+country_event = {
+	id = 8020439
+	title = "The Muggia Agreement activation"
```

Comment:
I think "The Muggia Agreement" would be better

Reply:
Same name as the earlier event?

Reply:
I think it is fine so long as 'activation' is not in the title as that makes it sound weird to a 3P.